include/swift/Option/Options.td

```diff
@@ -728,7 +728,12 @@ def disable_bridging_pch : Flag<["-"], "disable-bridging-pch">,
 def disable_print_package_name_for_non_package_interface :
   Flag<["-"], "disable-print-package-name-for-non-package-interface">,
   Flags<[FrontendOption, NoDriverOption, ModuleInterfaceOption, HelpHidden]>,
-  HelpText<"Disable adding package name to public or private interface">;
+  HelpText<"Deprecated, has no effect. Package name is by default only printed in package swiftinterface.">;
+
+def print_package_name_in_non_package_interface :
+  Flag<["-"], "print-package-name-in-non-package-interface">,
+  Flags<[FrontendOption, NoDriverOption, ModuleInterfaceOption, HelpHidden]>,
```

Comment:
this flag itself shouldn't be a `ModuleInterfaceOption`.

Reply:
This PR no longer contains this flag (will be introduced in the future instead).